lib/builders/restoreArgBuilder.js

```diff
@@ -4,7 +4,13 @@ module.exports = (value) => {
         return args;
     }
     if (value.source) {
-        args = args.concat(['--source', value.source]);
+        if (Array.isArray(value.source)) {
```

Comment:
`_.isArray(value.source)` if you don't mind. Not that this is incorrect, but I'd prefer to use lodash for most array functions